experiments/Train.ipynb

```diff
@@ -150,9 +150,9 @@
     "PREDICT_BATCH_SIZE = 128\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    TEST_SPLIT = 'train[90%:]'
</pre><p>Oh my god, this is so simple !</p><p>The only thing is that this is not random at all and you may have a big bias in the data.</p><p>I think at least of two important bias, linked to the fact that data is sorted by code barre numbers:</p><ol><li>you will have all french data (most qualified) at some point, with code starting with "3"</li><li>you will have dirty data at the beginning of the dataset</li></ol><p>So <strong>it's really mandatory to shuffle data.</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openfoodfacts/off-category-classification/pull/67/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Yes, tfds makes it quite easy to sample data! There are much more elaborate ways to split described at https://www.tensorflow.org/datasets/splits.

And that's why I was asking about shuffling before :) I'll switch to [`disable_shuffling=False`](https://github.com/openfoodfacts/off-category-classification/blob/master/datasets/off_categories/off_categories.py#L52) now, and will check whether that shuffle is deterministic (I expect it is, but better check).

Reply:
Deterministic shuffle confirmed here: https://github.com/tensorflow/datasets/issues/3068#issuecomment-787407626

Reply:
Really cool stuff to know. That's smart, I like it :-)